blocks/accordion/accordion.css

```diff
@@ -68,18 +75,18 @@
   .accordion details[open] .accordion-item-body details.child-accordion-item[open] .child-accordion-item-body p a {
     text-transform: uppercase;
     font-weight: bold;
-    color: #666;
+    color: var(--dark-gray-2);
     font-family: FrutigerBold, Arial, serif;
```

Comment:
Please use the body text var here.  We are using Open Sans now instead of Frutiger.